js/bybit.js

```diff
@@ -2198,8 +2533,8 @@ module.exports = class bybit extends Exchange {
             'timestamp': timestamp,
             'datetime': this.iso8601 (timestamp),
             'symbol': symbol,
-            'order': this.safeString2 (trade, 'order_id', 'orderId'),
-            'type': this.safeStringLower (trade, 'order_type'),
+            'order': this.safeString (trade, 'orderId'),
+            'type': this.safeStringLower (trade, 'orderType'),
```

Comment:
@sc0Vu bybit returns `unknown` for the order type, I think we should set it undefined in these cases: 
![image](https://user-images.githubusercontent.com/43336371/204820536-21a7f464-81a0-42c7-b053-52c40685344e.png)